tests/functional/test_new_resolver.py

```diff
@@ -228,6 +226,32 @@ def test_new_resolver_installs_extras(tmpdir, script, root_dep):
     assert_installed(script, base="0.1.0", dep="0.1.0")
 
 
+def test_new_resolver_installs_extras_deprecated(tmpdir, script):
```

Comment:
Why is this test in `test_new_resolver.py`? It's not explicitly resolver-related, is it? It should probably go in the main test suite.

Reply:
Also, do you not need to test use of the syntax on the command line, as well as in a requirements file?

Reply:
command line test should probably go in test/functional/test_install_extras.py... I 'll do that

As to the resolver-related question: It looks like I added a test but really it's an existing test I moved from one function to a new one. Many of the resolver wheel-install tests do look like generic tests (at least to an outside observer) but maybe moving those is another PR?

Reply:
Ah, right. It's fine here in that case - yes, these are pretty generic tests, and at some point will need to be moved into the main test suite. That's why I was concerned about adding more here that we didn't need to.

But your logic's sound. Leave it here and we can move it when we tidy up the others.